hedera-node/hedera-mono-service/src/test/java/com/hedera/services/contracts/execution/CreateEvmTxProcessorTest.java

```diff
@@ -127,8 +129,8 @@ void setup() {
 
     @Test
     void assertSuccessfulExecution() {
-        givenValidMock(true);
-        givenSenderWithBalance(350_000L);
+        givenValidMock(true, 350_000L, true);
+        //        givenSenderWithBalance(350_000L);
```

Comment:
Q: why all the commented out checks in this file.
Are the lines causing failures or are they no long applicable?
Should remove anything that is not applicable but we should be trying to preserver or improve test coverage